numerai-tournament/benchmark_models.md

```diff
@@ -0,0 +1,200 @@
+
+# ベンチマークモデル
+
+## 概要
+
+ベンチマークモデルは、Numeraiチームが構築した一連の標準モデルです。その予測は、毎ラウンド配布されるため、誰でも簡単にそれらを提出し、望むならステークすることができます。
+
+モデルのリストとその最近のパフォーマンスはこちらです：
+
+[NUMERAI BENCHMARK MODELS](https://numer.ai/~benchmark_models)
+
+## 予測のダウンロード
+
+検証およびライブの予測は[Numerai API](https://github.com/uuazed/numerapi)を介して利用できます。
+
+```python
+!pip install numerapi
+from numerapi import NumerAPI
+napi = NumerAPI()
+napi.download_dataset("v4.3/train_benchmark_models.parquet", "train_benchmark_models.parquet")
+napi.download_dataset("v4.3/validation_benchmark_models.parquet", "validation_benchmark_models.parquet")
+napi.download_dataset("v4.3/live_benchmark_models.parquet", "live_benchmark_models.parquet")
+```
+
+## ベンチマークモデルの作成方法
+
+### ウォークフォワード交差検証
+
+すべての予測はウォークフォワードフレームワークを使用して行われます。これは、すべての予測が、予測が行われる日付より前に利用可能なデータのみを使用してトレーニングされたモデルを使用して行われることを意味します。
+
+具体的には、データは156Eraのチャンクに分割されます。それから、各Eraチャンクごとに、最初のEraチャンクからパージEraまでトレーニングされたモデルによって予測が行われます。パージEraの数は、20Dターゲットの場合は8であり、60Dターゲットの場合は16です。
+
+したがって、Era157から313はEra148までトレーニングされたモデルを使用して予測され、Era314-470は306までトレーニングされたモデルを使用して予測されます。以降も同様です。
```

Comment:
We have updated this section to be more clear:
https://docs.numer.ai/numerai-tournament/models#how-they-made

Could you update this section to match please?

Reply:
I added this following the updates in the English version. Thanks for your review!